### PR TITLE
https://github.com/plante-app-team/plante/issues/85

### PR DESCRIPTION
**Pull request by cyclomaticcomplexity1981**
filters new design

Not a fan of the result:

https://user-images.githubusercontent.com/65116070/148696436-f2d0b00a-a090-48ca-b630-c5a777266347.mp4

https://user-images.githubusercontent.com/65116070/148696450-d3273d2b-0e2a-4040-acd1-2facf5ae4203.mp4

* the buttons are big, the user doesn't have an overview of the options(has to scroll)
* changing a filter also requires a lot of scrolling

I like the previous version better.

### CODE REVIEW

**Comment by cyclomaticcomplexity1981** [comment]
wait, if i get this story right the filters will be down to 3 buttons? https://github.com/plante-app-team/plante/issues/66
* all shops
* shops with products
* shops without products

then it might look work better

**Comment by blazern** [comment]
> wait, if i get this story right the filters will be down to 3 buttons? #66

Yes!

**Comment by blazern** [comment]
Haven't looked through the code yet, but I have a question regarding the videos - do you think it's possible make it so that the right and left sides of the horizontal list aren't chopped off?

I know searchbar and other top things are children of a `Padding` with `left` and `right` ==24.
Maybe instead of:
```
Padding(
            padding: const EdgeInsets.only(left: 24, right: 24, top: 44),
            child: Column(children: [
              ...
              MapHintsList(controller: _hintsController),
              Consumer(
                  builder: (context, ref, _) => AnimatedMapWidget(
                      child: _mode.watch(ref).buildTopActions())),
              ...
            ])),
```
...we could have:
```

Padding(
            padding: const EdgeInsets.only(top: 44),
            child: Column(children: [
              ...
              // Each child now has horizontal padding == 24 (except for _mode.watch(ref).buildTopActions()),
              // the Padding parent doesn't have a horizontal padding
              Padding(
                     padding: const EdgeInsets.only(left: 24, right: 24),
                     child: MapHintsList(controller: _hintsController)),
              Consumer(
                     builder: (context, ref, _) => AnimatedMapWidget(
                     child: _mode.watch(ref).buildTopActions())),
              ...
```
?

Or something similar but shorter and nicer (not sure if "shorter and nicer" is possible).

**Comment by cyclomaticcomplexity1981** [comment]
I'll have a look. Well since the number of buttons is going down to 3 Some other implementations will change too.

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -179,7 +180,7 @@
     "map_page_location_permission_go_to_settings": "Turn On in Settings",
     "map_page_location_permission_reasoning_settings": "Plante works best with Location Services turned on. You'll get information about stores based on your location",
     "map_page_location_permission_title": "Location Services",
-    "map_page_no_shops_hint2": "No stores with products nearby. You can tune stores filters by clicking",
+    "map_page_no_shops_hint2": "There are no stores in the area yet. Try moving the map, add a new store or try the map filters.",
```
Comment (blazern) [inline lib/l10n/app_en.arb:183]: The string ID probably should be changed to `map_page_no_shops_hint3` since new translations will 100% be needed :)

**Comment by blazern** [comment]
Should I wait for
> wait, if i get this story right the filters will be down to 3 buttons?

?

**Comment by blazern** [comment]
The code compiles and the app starts.
The checks above fail because of an extra import.

**Pull request merged**